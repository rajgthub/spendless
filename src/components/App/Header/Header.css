.header{
    height: 20vh;
    background-color: #4BBA00;
    color: #FFFFFF;
}