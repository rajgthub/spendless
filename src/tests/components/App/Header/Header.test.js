import React from "react";
import {shallow} from 'enzyme'
import Header from "../../../../components/App/Header";
test("should render Header ", () => {
  const wrapper = shallow(<Header />)
  expect(wrapper).toMatchSnapshot()
  const logoMsg = <h1>SpendLess</h1>
  expect(wrapper.contains(logoMsg)).toBe(true)
});
