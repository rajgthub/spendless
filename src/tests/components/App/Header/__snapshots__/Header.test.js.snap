// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header  1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        SpendLess
      </h1>,
      "className": "header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "SpendLess",
      },
      "ref": null,
      "rendered": "SpendLess",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          SpendLess
        </h1>,
        "className": "header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "SpendLess",
        },
        "ref": null,
        "rendered": "SpendLess",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
